src/documents/file_handling.py

```diff
@@ -158,18 +159,22 @@ def generate_filename(doc, counter=0, append_gpg=True, archive_filename=False):
             else:
                 asn = "none"
 
+            # Convert UTC database date to localized date
+            local_added = timezone.localdate(doc.added)
+            local_created = timezone.localdate(doc.created)
```

Comment:
The code before dealt with the situation, that `doc.created` probably doesn't exist, same for `doc.added`. How do you now deal with that situation?

Reply:
It kind of dealt with it, but didn't really, so I no longer do.  It weirdly only checked that for the year, month or day, but not when using the date as a whole.  So if a date could not exist already, there would have been reports of it,

For both `.created` and `.added`, the field definition doesn't include `null=True`, so I don't think it's possible to get these fields to not exist.  They also have a default set.

Reply:
Understood. Thought already what the scenario would be, in which added / created do not exist...